src/navigation/failure_identification/watchdog.py

```diff
@@ -1,11 +1,98 @@
 import numpy as np
 import rospy
 from pandas import DataFrame
+from util.ros_utils import get_rosparam
+from util.SO3 import SO3
+from typing import Tuple
+
+WINDOW_SIZE = get_rosparam("watchdog/window_size", 100)
+ANGULAR_THRESHOLD = get_rosparam("watchdog/angular_threshold", 0.001)
+LINEAR_THRESHOLD = get_rosparam("watchdog/linear_threshold", 0.55)
 
 
 class WatchDog:
-    def __init__(self):
-        pass
+    def __init__(self, collector_in):
+        self.collector = collector_in
```

Comment:
maybe I'm missing some thing, but is this actually used?

Reply:
no its not I removed it